### PR TITLE
Change identifier from 'name' to 'id' for PCI Compliance

### PR DESCRIPTION
**Pull request by Islati**
While attempting integration with Stripe using this component I noticed there was an issue regarding the name attribute being specifically requested.

Another user (@sylv3r) previously forked the repo with a fix to integrate the proper usage.

I simply integrated them with CJSX for build, updated the example, and forth.

I believe it to be highly beneficial for the project and its users, as most integrate this interface with an alternate service as opposed to their own PCI Compliant integrations.

### CODE REVIEW

File: src/components/card-react-form-container.cjsx [hunk 1]
```diff
@@ -85,16 +86,18 @@ CardReactFormContainer = React.createClass
       if typeof child isnt 'object' or child is null
         return child
 
-      if (child.props and child.props.name)
+      inputAttribute = child.props[@props.inputAttributeIndentifier]
+
+      if (child.props and child.props.id)
```
Comment (shatran) [inline src/components/card-react-form-container.cjsx]: @TechnicalBro isn't this forcing the attribute to be id?

File: lib/card-react-form-container.js [hunk 2]
```diff
@@ -84,17 +85,18 @@
...
       return React.Children.map(children, (function(_this) {
         return function(child) {
-          var inputsValidationClass, newClassName;
+          var inputAttribute, inputsValidationClass, newClassName;
           if (typeof child !== 'object' || child === null) {
             return child;
           }
-          if (child.props && child.props.name) {
+          inputAttribute = child.props[_this.props.inputAttributeIndentifier];
+          if (child.props && child.props.id) {
```
Comment (shatran) [inline lib/card-react-form-container.js]: @TechnicalBro same here, isn't this forcing the attribute to be id?

**Review by shatran — CHANGES_REQUESTED** [review]

**Comment by sylver** [comment]
Thanks for the credit @TechnicalBro
Indeed, my fork wasn't ready for a PR yet, nice work with this one.

I also noticed this *huge* problem which is basically making this component useless for submitting sensible informations like credit card data ... which is actually its main purpose !

I strongly encourage to merge this PR asap.

File: lib/card-react-form-container.js (hunk 2 above)
Comment (sylver) [reply lib/card-react-form-container.js]: @shatran @TechnicalBro indeed, should be
`if (child.props && inputAttribute) {`
That's a typo I guess ;)

File: src/components/card-react-form-container.cjsx (hunk 1 above)
Comment (sylver) [reply src/components/card-react-form-container.cjsx]: same here 
`if (child.props && inputAttribute) {`
Nice shot @shatran

File: lib/card-react-form-container.js (hunk 2 above)
Comment (Islati) [reply lib/card-react-form-container.js]: I'll merge this up right away- Tossed it together pretty quick :)

**Comment by Islati** [comment]
Thanks for the heads up @shatran, @sylv3r !

That variable has been slotted in and should be all that's required to have the identifier specified as part of the form container now!

Hope this helps!

**Pull request closed without merging**